src/chains/ethereum/console.log/README.md

```diff
@@ -49,7 +49,7 @@ import { ConsoleLogs, maybeGetLogs } from "@ganache/console.log";
 // ... initialize vm ...
 
 // vm should be an instance of @ethereumjs/vm
-vm.on("step", event => {
+vm.evm.on("step", event => {
```

Comment:
```suggestion
vm.evm.events.on("step", event => {
```
By the way, @davidmurdoch, is this a breaking change?

Reply:
No, this isn't our VM. The example requires the use provider their own instance.